frontend/src/metabase/admin/settings/notifications/WebhookForm.tsx

```diff
@@ -0,0 +1,212 @@
+import type { FormikHelpers } from "formik";
+import { t } from "ttag";
+import * as Yup from "yup";
+
+import { useTestChannelMutation } from "metabase/api/channel";
+import {
+  Form,
+  FormChipGroup,
+  FormProvider,
+  FormSubmitButton,
+  FormTextInput,
+} from "metabase/forms";
+import { FormKeyValueMapping } from "metabase/forms/components/FormKeyValueMapping";
+import { useActionButtonLabel } from "metabase/hooks/use-action-button-label";
+import { Button, Chip, Flex, Alert, Text, Group, Icon } from "metabase/ui";
+import type { NotificationAuthMethods } from "metabase-types/api";
+
+const validationSchema = Yup.object({
+  url: Yup.string()
+    .url(t`Please enter a correctly formatted URL`)
+    .required(),
+  name: Yup.string().required(t`Please add a name`),
+  description: Yup.string().required(t`Please add a description`),
```

Comment:
does description really have to be required?

Reply:
The API requires it, but yeah, I think that can be adjusted